src/System.Private.ServiceModel/tests/Scenarios/Client/ClientBase/DuplexClientBaseTests.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.ServiceModel;
+using System.ServiceModel.Channels;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Xunit;
+
+public static class DuplexClientBaseTests
+{
+    static TimeSpan s_WaitTimeout = TimeSpan.FromSeconds(10);
+
```

Comment:
Nit: in future we may want this to have a more descriptive name ("MaxTestWaitTime?") and move it into the common scenarios folder.  I suspect we will use it again.  We can wait for the 2nd use before moving it.